src/lib/protobuf-c/protobuf-c.c

```diff
@@ -45,8 +45,8 @@
  * \todo Use size_t consistently.
  */
 
-#include <stdlib.h>	/* for malloc, free */
 #include <string.h>	/* for strcmp, strlen, memcpy, memmove, memset */
+#include "osal_common.h"	/* for malloc, free */
```

Comment:
Need to update this appropriately after concluding osal file/dir structure mentioned in the previous comment.